pipeline/config-dev.groovy

```diff
@@ -253,7 +253,7 @@ app {
 environments {
     'dev' {
         vars {
-            DB_PVC_SIZE = '1Gi'
+            DB_PVC_SIZE = '10Gi'
```

Comment:
Remember to resize these environment variables to the appropriate sizes when you are done testing.

Reply:
Yeah I was gonna revert them when I knew the PR was approved